src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
```

Comment:
[CODE STYLE] - The variable name `result` is not descriptive enough and could decrease code readability. Consider using a more descriptive name, such as `stylesObject` or `cssProperties`.